ops/environments/000_cross_env_vars

```diff
@@ -0,0 +1,139 @@
+# Anistrano
+ansistrano_deploy_from: "{{ playbook_dir }}/../" # Where my local project is (relative or absolute path)
+ansistrano_deploy_to: "/opt/{{ project_name }}" # Base path to deploy to.
+ansistrano_version_dir: "releases" # Releases folder name
+ansistrano_current_dir: "current" # Softlink name. You should rarely changed it.
+ansistrano_current_via: "symlink"
+ansistrano_keep_releases: 3 # Releases to keep after a new deployment. See "Pruning old releases".
+ansistrano_shared_paths:
+  - log
+  - public/assets
+  - public/system
+  - tmp/cache
+  - tmp/derivatives
+  - tmp/pids
+  - tmp/sockets
+  - tmp/uploads
+  - vendor/bundle
+
+ansistrano_shared_files:
+  - config/database.yml
+  - config/secrets.yml
+  - config/fedora.yml
+  - config/blacklight.yml
+  - config/redis.yml
+  - config/resque-pool.yml
+  - config/solr.yml
+
+ansistrano_deploy_via: "git"
+#ansistrano_git_repo: https://github.com/ucsdlib/horton.git
+ansistrano_git_repo: https://github.com/notch8/horton.git
+ansistrano_git_branch: devops # What version of the repository to check out. This can be the full 40-character SHA-1 hash, the literal string HEAD, a branch name, or a tag name
+#ansistrano_git_identity_key_path: "keys/deploy" # If specified this file is copied over and used as the identity key for the git commands, path is relative to the playbook in which it is used
+ansistrano_before_symlink_tasks_file: "{{ playbook_dir }}/roles/deploy/tasks/before_symlink.yml"
+ansistrano_after_symlink_tasks_file: "{{ playbook_dir }}/roles/deploy/tasks/after_symlink.yml"
+
+capistrano_user: deploy
+capistrano_group: deploy
+deploy_target: development
+
+# Database
+db: postgres
+db_adapter: postgresql
+hydra_db: hydra
+hydra_db_user: horton
+hydra_db_pass: penguins;13f1
```

Comment:
these should be jinja variables too?

Reply:
@mcritchlow I'm not sure I understand what you mean here. this file is full of config variable for the various parts.  These db creds specifically are the defaults for the dev environment.  they are overridden by an encrypted config file which has a few prod specific things in it.   Here are the keys from that prod file (values left out)


# Prod items
proxy_env
  http_proxy
  https_proxy


# Database
db
db_adapter
hydra_db
hydra_db_user
hydra_db_pass
hydra_db_host
hydra_db_port
fedora_db
fedora_db_user
fedora_db_pass
fedora_db_host
fedora_db_port

deploy_target

# Postfix
postfix_relayhost
postfix_smtp_sasl_auth_enable
postfix_smtp_sasl_user
postfix_smtp_sasl_password
postfix_send_test_mail_to
postfix_rewrite_sender_address

Reply:
Ah, yes, I was reading this wrong. Thanks for clarifying :+1: